recipe/meta.yaml

```diff
@@ -56,39 +70,38 @@ requirements:
     - gts
     # --with-gdk-pixbuf=yes
     - gdk-pixbuf                                          # [unix]
-    # --with-gdk=yes
-    - gtk2                                                # [unix]
-    # pangocairo requires glib
-    - glib
     # --with-libgd=yes
     - libgd
+    # --with-gdk=yes
+    - gtk2                                                # [linux]
     # --with-rsvg=yes
     - librsvg                                             # [unix]
     # --enable-ltdl
     - libtool                                             # [unix]
     # --with-webp=yes
     - libwebp-base                                        # [unix]
     # --with-pangocairo=yes
+    - cairo
     - pango
+    - glib
     # default requirement
     - zlib                                                # [unix]
-    # Windows only deps
     - getopt-win32                                        # [win]
-
   run:
     # A few satisfy needs via run_exports, rest rely on pin_run_as_build
     # Add all of them anyway ¯\_(ツ)_/¯
-    - cairo
     - expat                                               # [unix]
     - gts
-    - gtk2                                                # [unix]
     - libgd
-    - glib
+    - gtk2                                                # [linux]
     - librsvg                                             # [unix]
```

Comment:
I think we need lirsvg >=2.36.0, see https://gitlab.com/graphviz/graphviz/-/blob/main/CMakeLists.txt#L207